apps/inviwo/CMakeLists.txt

```diff
@@ -173,3 +173,12 @@ endif()
 # Save information for python tools.
 ivw_mod_name_to_class(ivw_module_classes ${enabled_modules})
 ivw_private_create_pyconfig("${IVW_MODULE_DIR};${IVW_EXTERNAL_MODULES}" "${ivw_module_classes}" inviwo)
+
+
+# Windows specific build steps
```

Comment:
What is this? Why is it here?

Reply:
My bad, was testing some stuff, must have accidentally added it to the commit